datasets/verkiezingen/verkiezingen.json

```diff
@@ -0,0 +1,53 @@
+{
+  "type": "dataset",
+  "id": "verkiezingen",
+  "title": "Verkiezingen",
+  "status": "beschikbaar",
+  "description": "Gegevens over verkiezingen",
+  "version": "0.0.1",
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "processenverbaal",
+      "title": "Processenverbaal",
+      "description": "URI en metadata processenverbaal in pdf per jaar, per stemlocatie",
+      "type": "table",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["schema", "id"],
+        "display": "jaar",
```

Comment:
Is jaar als display veld wel onderscheidend genoeg? Ik zie bij de DAG dat er blijkbaar meerdere records binnengehaald worden, die zullen dan toch hetzelfde jaar hebben?